kbuild/utils/README.md

```diff
@@ -51,6 +51,17 @@ In the directory, you can find:
    * `make`
 5. The image can be found in `output/images`
 
+## Editing or tweaking the image
+
+To make simple changes to the image:
+
+1. `enkit astore get $uid` or `enkit astore get $path` to download the image.
+2. `apt-get install fuse2fs` if not already installed on your machine.
+3. `fuse2fs ./rootfs.img /mnt/tmp`
```

Comment:
Should we keep a changelog of what changed in the image? Without one, we wouldn't know what went into the astore image. Maybe an astore object comment with each push (so include the command to use in step #6)?

Reply:
Added a step 6 here, good point. Apologies, accidentally force pushed an amend.